resources/js/components/FormField.vue

```diff
@@ -137,6 +167,15 @@ export default {
     },
   },
 
+  mounted() {
+    var diff = this.field.minRows - this.fieldsWithValues.length;
+    if (this.canAdd() || Object.keys(this.fieldsWithValues).length === 0 || diff > 0) {
```

Comment:
Miks siin `Object.keys(this.fieldsWithValues).length` on? `this.fieldsWithValues.length` töötab ka :D

Reply:
Niisama xd.